internal/querywatcher/query_watcher.go

```diff
@@ -121,35 +125,55 @@ func (w *QueryWatcher) watchKeys(ctx context.Context, watchChan <-chan dstore.Wa
 }
 
 // processWatchEvent processes a single watch event.
-func (w *QueryWatcher) processWatchEvent(event dstore.WatchEvent) {
-	w.WatchList.Range(func(key, value interface{}) bool {
-		query := key.(DSQLQuery)
-		clients := value.(*sync.Map)
-
-		if !regex.WildCardMatch(query.KeyRegex, event.Key) {
+func (w *QWatchManager) processWatchEvent(event dstore.WatchEvent) {
+	/*
+		Iterate over the watchlist to go through all fingerprints and the corresponding client connections to those fingerprints
+	*/
```

Comment:
nit: shouldn't we use `//` style comments here? I am not familiar with what the recommend styling guide is for comments in golang, so I'm happy to be corrected here.

Reply:
We should use // types comments in go

Reply:
Fixed this.